cypress/integration/acceptance/user-activity.specs.js

```diff
@@ -0,0 +1,256 @@
+'use strict';
+
+const moment = require('moment');
+const testConfig = require('../../../test/acceptance/config');
+const AuditSearchPage = require('../../pages/audit/AuditSearchPage');
+const LoginPage = require('../../pages/LoginPage');
+const env = testConfig.env;
+
+const searchNoRecord = {
+  from: '01/01/1800',
+  to: '04/01/1800',
+  user: ''
+};
+
+describe('User Activity', () => {
+  const user = env !== 'local' ? testConfig.username : 'lev-e2e-tests';
```

Comment:
env is loaded by cypress in the cypress/plugins folder. The correct way to obtain a reference is using Cypress.env('env') which will give you 'local', or Cypress.env('e2e') which will give you a boolean.

Reply:
Amended